mlir/test/rocmlir-driver/populate.mlir

```diff
@@ -1,7 +1,7 @@
-// RUN: rocmlir-gen --arch %arch -p -mfma=off -atomic_add=off -atomic_fmax_f32=off -dot=on | FileCheck %s --check-prefix=F32
```

Comment:
`-mfma=off -wmma=off -atomic_add=off` seems to be redundant as they should by default be off?

Reply:
Nah, if `%arch` supports them, they'll be on